testing/test_runners/TestLibraryCode_Runner.c

```diff
@@ -0,0 +1,8 @@
+#include "unity.h"
+#include "unity_fixture.h"
+
+TEST_GROUP_RUNNER(LibraryCode)
```

Comment:
Does this have to be a separate file from the main runner?

Reply:
Short answer: No it does not have to be.
Long answer: It will make it much easier for us in the future when we add in more texts and this goes by unity's standards for implementing their testing framewok